.github/workflows/ci.yml

```diff
@@ -29,6 +29,8 @@ jobs:
 
       - name: Compile and Test
         run: |
+          mkdir config
+          echo "import Config\nconfig :adbc, drivers: [:sqlite]" > config/config.exs
```

Comment:
For CI, I think we can do `Adbc.Driver.download(...)` in the `test_helper.exs`, no?

Reply:
Oh that should be okay I think?